Dockerfile-web

```diff
@@ -8,6 +8,7 @@ RUN dotnet publish Minitwit.Web -c Release -o /Release
 
 EXPOSE 8080
 ENV ASPNETCORE_URLS=http://*:8080
+ENV ASPNETCORE_ENVIRONMENT=Development
```

Comment:
We should not do this as the Dockerfile is used in production and the production *is not* development.